README.md

```diff
@@ -0,0 +1,58 @@
+# IntelliJ Kubernetes
+
+## Overview
+
+JetBrains IntelliJ plugin for interacting with Kubernetes and OpenShift clusters.
+The plugin tries to mimic the vscode-kubernetes-tools extension that exists for vscode.
+This plugin is currently in Preview Mode.
+
+![](images/demo1.gif)
+
+### Kubernetes & OpenShift resource tree
+The plugin offers a tool window with a tree that displays all the resources that exist on a Kubernetes or OpenShift cluster.
+The tree displays all the contexts that exist in your kube config (~/.kube/config).
+The tree is dynamically updated when external tools like kubectl, oc, etc. change the kube config. 
+
+#### Current Context
+The tree displays resources that exist on the cluster that the current context points to. 
+The user can switch the current context and cluster by choosing any context that's listed and picking "Set as Current Cluster" in the context menu.
+OpenShift clusters are marked as such with a OpenShift icon instead of a Kubernetes icon.
+ 
+#### Current Namespace/Project
+The tree displays resources that exist within the current namespace as specified in the kube config. 
+Resources that are not bound to a namespace/project are also listed.
```

Comment:
Maybe these two sentences can be merged. `The tree displays resources that exist within the current namespace/project and that are not owned to any namespace/project.`